x/auth/client/cli/tip.go

```diff
@@ -0,0 +1,84 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	clienttx "github.com/cosmos/cosmos-sdk/client/tx"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+)
+
+func GetTipsToFeeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tips-to-fee <aux_signed_tx.json>",
+		Short: "tips to fee broadcast the aux signed tx, and sends the tip amount to the broadcaster",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			auxTx, err := authclient.ReadTxFromFile(clientCtx, args[0])
+			if err != nil {
+				return err
+			}
+
+			tipTx, ok := auxTx.(tx.TipTx)
+			if !ok {
+				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "invalid transaction")
+			}
+
+			// b, err := clientCtx.TxConfig.WrapTxBuilder(tipTx)
+			// if err != nil {
+			// 	return err
+			// }
+
+			// b.AddAuxSignerData()
```

Comment:
is there any way to get aux signer data(`GetAuxSignerData`) here?
@amaurym , @aaronc

Reply:
With the new AuxTxBuilder, the flow is this:
1. user1 (aux signer) uses the AuxTxBuilder to create an AuxSignerData
2. user1 saves AuxSignerData in a `aux.json` file
3. user2 uses this `tips-to-fee` command, takes in the `aux.json`

So:
- in L26, we should not use ReadTxFromFile. Maybe we can create a `readAuxSignerDataFromFile`, which reads the .json file, and unmarshals it into a AuxSignerData
- L31, we don't cast to `tx.TipTx` anymore, since the input is already a AuxSignerData

Reply:
I need to think about how we do step1. Maybe we need another flag.